drupal/code/modules/features/pori_configuration/pori_configuration.module

```diff
@@ -21,3 +21,25 @@ function pori_configuration_language_switch_links_alter(&$links, $type, $path) {
     }
   }
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function pori_configuration_form_views_exposed_form_alter(&$form, &$form_state, $form_id) {
+  if ($form['#id'] == 'views-exposed-form-kada-admin-content-page') {
+    global $user;
```

Comment:
This and following line seem redundant. global user return current user object and the following line just reloads it.

Reply:
It feels there must be some reason to that but @teroelonen feel free to fix later if not. It feels there must be some reason for this so approving this now.